server/supernova/audio_backend/portaudio_backend.hpp

```diff
@@ -285,18 +293,25 @@ class portaudio_backend : public detail::audio_backend_base<sample_type, float,
             super::fetch_inputs(inputs, blocksize_, input_channels);
             engine_functor::run_tick();
             super::deliver_outputs(outputs, blocksize_, m_hwOutputChannels);
+            if (IsClipping)
+                for (uint16_t i = 0; i != m_hwOutputChannels; ++i)
+                    if (super::is_aligned(out[i]))
```

Comment:
`is_aligned` check is a copy-paste from https://github.com/supercollider/supercollider/blob/28ff671ea25f1bb98ce7e712cb6f3c324d349fb4/server/supernova/audio_backend/audio_backend_common.hpp#L101 I have a very vague understanding what this is doing, I'm looking for any feedback about this.